packages/tsutils/src/typeGuards.ts

```diff
@@ -8,5 +8,13 @@ export const isDefined = <T>(value: T): value is Exclude<T, undefined> => value
 export const isNotNullish = <T>(val: T): val is Exclude<T, undefined | null> =>
   val !== undefined && val !== null;
 
+export const ensure = <T>(val: T) => {
```

Comment:
My only concern is how was this code functioning before these manual checks were added?

Shouldn't tsc have complained of a possibly nullish value? 
Are the types on our @tupaia/database functions wrong?

Reply:
There was at least one case where we'd let an `any` slip in, and from then on anything that dealt with that object was using `any` type. So we basically didn't have type safety. I think it was the fetch from `entityApi.getAncestorsOfEntity()`.

Reply:
Gonna merge this in so Raj can start testing, but happy to chat on the naming more if ya want